### PR TITLE
os: add section on visudo usage to adding-users doc

### PR DESCRIPTION
**Pull request by kbrwn**
Visudo on CoreOS requires the file be specified or an error is thrown.
Issue: https://github.com/coreos/bugs/issues/1617

### CODE REVIEW

File: os/adding-users.md [hunk 1]
```diff
@@ -64,6 +64,33 @@ To assign an SSH key, run:
 update-ssh-keys -u user1 user1.pem
 ```
 
+## Grant sudo Access
+
+If you trust the user you can grant administrative privileges using visudo. visudo checks the file syntax before actually overwriting the sudoers file. This command should be ran as root to avoid losing sudo access in the event of a failure. Instead of editing /etc/sudo.conf directly you will create a new file under the `/etc/sudoers.d/` directory. When you run visudo it is required that you specify which file you are attempting to edit with the -f argument:
```
Comment (joshix) [inline os/adding-users.md]: backticks: `visudo`.
backticks: `sudoers`

Comment (crawford) [inline os/adding-users.md]: "If you trust the user, you..."

Comment (crawford) [inline os/adding-users.md]: "This command should be run as root..."

Comment (crawford) [inline os/adding-users.md]: "When you run visudo, it..."

**Comment by crawford** [comment]
Can you please rebase before merging?

**Pull request merged**